### DISCUSSION TITLE
Is YTUHD still needed?

### DISCUSSION THREAD

**Discussion by plain-mame8** [qnblackcat/uYouPlus#352] [Q&A] [answered]
I am a newbie in this community so sorry if this is a silly question.

The official youtube app have been enable to play videos at 1440p or higher. 
Do you still need YTUHD which is still unstable?

**Comment by qnblackcat** [comment, marked as answer]
Short answer: Yes, YTUHD is definitely needed.

Long answer: Apple literally gave YouTube a private entitlement to use VP9 codec (which is needed for 2K/4K) quality. Because it is a private entitlement, no one can use it but Google, which means no VP9 codec when sideloading YouTube. Without YTUHD, we won't be able to see 2K/4K quality on sideloaded YouTube.

**Reply by plain-mame8** [reply]
I understood it very well.
Thank you.
